modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PendingCommerceOrderTableFDSView.java

```diff
@@ -45,8 +47,7 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 		).add(
 			"purchaseOrderNumber", "purchase-order-number"
 		).add(
-			"date", "create-date",
-			fdsTableSchemaField -> fdsTableSchemaField.setSortable(true)
+			_addDateFDSTableSchemaField()
```

Comment:
we must delegate creation of this field to a separate method as builder manages a more abstract interface